ddtrace/contrib/redis/util.py

```diff
@@ -41,7 +41,10 @@ def format_command_args(args):
     out = []
     for arg in args:
         try:
-            cmd = stringify(arg)
+            try:
+                cmd = ensure_text(arg, errors="backslashreplace")
+            except TypeError:
+                cmd = stringify(arg)
```

Comment:
For here we could do:

```python
if isinstance(arg, (six.binary_type, six.text_type)):
    cmd = ensure_text(arg, errors="backslashreplace")
else:
    cmd = stringify(arg)
```

What do you think?

https://github.com/benjaminp/six/blob/45f1a230f9cc8e48372e19627b91ac06a2013292/six.py#L939-L955

Reply:
Yeah, given that `six` is unlikely to change `ensure_text` anytime in the foreseeable future, seems like it would be good to avoid the exception. Will fix it shortly.

Reply:
I took the liberty of adding `text_type` to `compat` to avoid directly referencing `six` types. The existing `string_type` would not have worked for Python 2 as it maps to `basestring` and `text_type` maps to `unicode`.

Happy to refer directly to the `six` types though.